iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/IotHubServicesCommon.java

```diff
@@ -55,10 +55,6 @@ public static void sendMessages(InternalClient client,
         try
         {
             client.open();
-            if (statusUpdates != null)
-            {
-                confirmOpenStabilized(statusUpdates, 120000, client);
```

Comment:
I see confirmOpenStabilized being referenced in waitForStabilizedConnection, is that required?

Reply:
Yes, that is used during our error injection tests so that the test waits for the error injection period to end